### PR TITLE
Introduction rework for updates and flow

### PR DESCRIPTION
**Pull request by michhar**
* Updated all notebooks for this delivery (people, announcements, etc.)
* Added notebook for basic PyTorch and tensors

### CODE REVIEW

File: Introduction/3.Introduction_to_Tensors.ipynb [hunk 1]
```diff
@@ -0,0 +1,478 @@
...
+    "compute their gradients, and run the back propagation algorithm. An\n",
+    "in-depth discussion of that algorithm is beyond the scope of this\n",
+    "tutorial."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Lets have Pytorch compute the gradient, and see that we were right:\n",
```
Comment (kendallc) [inline Introduction/3.Introduction_to_Tensors.ipynb]: PyTorch (capitalized T)

File: Introduction/1.Workshop_Introduction.ipynb [hunk 2]
```diff
@@ -29,15 +29,16 @@
...
-    "* Mehrdad Yazdani (January, Inc.) - Speaking about Character Level Classification\n",
-    "* Kendall Chuang (Ayasdi) - Introduction Word Embeddings\n",
-    "* David Clark (Fresh Gravity) - Speaking about LSTMs\n",
-    "* Micheleen - Bidirectional LSTMs, Conditional Random Fields\n",
-    "* Kendall Chuang - ULMFit and Closing"
+    "Speakers:\n",
+    "* Kendall Chuang (Carta)\n",
+    "* David Clark (Data Science Consultant)\n",
+    "* Priya Ananthasankar (Microsoft)\n",
+    "* Micheleen Harris (Microsoft)\n",
```
Comment (kendallc) [inline Introduction/1.Workshop_Introduction.ipynb]: Should we do this in talk order or alphabetical?

File: Introduction/1.Workshop_Introduction.ipynb [hunk 3]
```diff
@@ -59,10 +64,11 @@
     }
    },
    "source": [
-    "## Questions about you\n",
-    "* Python background?\n",
-    "* Jupyter background?\n",
+    "## Questions to you\n",
```
Comment (kendallc) [inline Introduction/1.Workshop_Introduction.ipynb]: nit: "for"

File: Introduction/2.Setup.ipynb [hunk 4]
```diff
@@ -96,13 +95,50 @@
...
+    "3.  Place a check in the box by the `nlp_workshop.zip` file and click \"Download\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "! zip -r nlp_workshop.zip ../../intro-to-nlp-with-pytorch"
```
Comment (kendallc) [inline Introduction/2.Setup.ipynb]: We should try this out on the Azure notebooks before the workshop

File: Introduction/3.Introduction_to_Tensors.ipynb [hunk 5]
```diff
@@ -0,0 +1,478 @@
...
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Introduction to PyTorch's Tensor Library\n",
+    "\n",
+    "Original Author:  Robert Guthrie (<a href=\"https://github.com/pytorch/tutorials/blob/master/beginner_source/nlp/pytorch_tutorial.py\" target=\"_blank\">Link</a>)\n",
+    "\n",
+    "All of deep learning is computations on tensors, which are\n",
```
Comment (kendallc) [inline Introduction/3.Introduction_to_Tensors.ipynb]: "is" --> "consists of"

File: Introduction/3.Introduction_to_Tensors.ipynb [hunk 6]
```diff
@@ -0,0 +1,478 @@
...
+    "First, think from a programmers perspective. What is stored in the\n",
+    "torch.Tensor objects we were creating above? Obviously the data and the\n",
+    "shape, and maybe a few other things. But when we added two tensors\n",
+    "together, we got an output tensor. All this output tensor knows is its\n",
+    "data and shape. It has no idea that it was the sum of two other tensors\n",
+    "(it could have been read in from a file, it could be the result of some\n",
+    "other operation, etc.)\n",
+    "\n",
+    "If ``requires_grad=True``, the Tensor object keeps track of how it was\n",
+    "created. Lets see it in action."
```
Comment (kendallc) [inline Introduction/3.Introduction_to_Tensors.ipynb]: "Let's"

File: Introduction/3.Introduction_to_Tensors.ipynb [hunk 7]
```diff
@@ -0,0 +1,478 @@
...
+    "s = z.sum()\n",
+    "print(s)\n",
+    "print(s.grad_fn)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "So now, what is the derivative of this sum with respect to the first\n",
```
Comment (kendallc) [inline Introduction/3.Introduction_to_Tensors.ipynb]: Note: might want to add "linear algebra/calculus" as an optional prereq for the workshop

File: Introduction/3.Introduction_to_Tensors.ipynb [hunk 8]
```diff
@@ -0,0 +1,478 @@
...
+    "\n",
+    "$$s = \\overbrace{x_0 + y_0}^\\text{$z_0$} + \\overbrace{x_1 + y_1}^\\text{$z_1$} + \\overbrace{x_2 + y_2}^\\text{$z_2$}$$\n",
+    "\n",
+    "And so s contains enough information to determine that the derivative\n",
+    "we want is 1!\n",
+    "\n",
+    "Of course this glosses over the challenge of how to actually compute\n",
+    "that derivative. The point here is that s is carrying along enough\n",
+    "information that it is possible to compute it. In reality, the\n",
+    "developers of Pytorch program the sum() and + operations to know how to\n",
```
Comment (kendallc) [inline Introduction/3.Introduction_to_Tensors.ipynb]: "PyTorch"

File: Introduction/3.Introduction_to_Tensors.ipynb (hunk 1 above)
Comment (kendallc) [reply Introduction/3.Introduction_to_Tensors.ipynb]: About gradients -- we might want to say we'll discuss why we need gradients/backprop/gradient descent somewhere, or say we will do it in a future part. We get into gradients here without saying why.

File: Introduction/2.Setup.ipynb (hunk 4 above)
Comment (michhar) [reply Introduction/2.Setup.ipynb]: Agreed.  Just set up the VM.

File: Introduction/3.Introduction_to_Tensors.ipynb (hunk 7 above)
Comment (michhar) [reply Introduction/3.Introduction_to_Tensors.ipynb]: Good point.  We can add this in the future.  I'll place a note that it's optional and only for reference in the notebook.

File: Introduction/1.Workshop_Introduction.ipynb [hunk 9]
```diff
@@ -221,7 +182,8 @@
     "## References\n",
     "1.  [RNN video \"RNN1. Why sequence models?\"](https://www.youtube.com/watch?v=5Vl-bK7tfD8&list=PLBAGcD3siRDittPwQDGIIAWkjz-RucAc7&index=1) by Andrew Ng\n",
     "2.  [Getting Started with PyTorch Part 1: Understanding how Automatic Differentiation works](https://towardsdatascience.com/getting-started-with-pytorch-part-1-understanding-how-automatic-differentiation-works-5008282073ec)\n",
-    "3.  [Introduction to PyTorch fro pytorch.org](https://pytorch.org/tutorials/beginner/nlp/pytorch_tutorial.html#sphx-glr-beginner-nlp-pytorch-tutorial-py)\n"
+    "3.  [Introduction to PyTorch fro pytorch.org](https://pytorch.org/tutorials/beginner/nlp/pytorch_tutorial.html#sphx-glr-beginner-nlp-pytorch-tutorial-py)\n",
```
Comment (kendallcarta) [inline Introduction/1.Workshop_Introduction.ipynb]: "fro" --> from or for?

**Comment by kendallcarta** [comment]
Looks good to me!
